Dockerfile

```diff
@@ -43,5 +43,4 @@ RUN git diff-index --cached --exit-code HEAD
 
 
 RUN govendor test -v -timeout 10s +local
-CMD cp -r henson /build/ && go build -a -v -ldflags "-X github.com/stripe/veneur.VERSION=$(git rev-parse HEAD)" -o /build/veneur ./cmd/veneur && \
-  cp -r henson /build/ && go build -a -v -ldflags "-X github.com/stripe/veneur-proxy.VERSION=$(git rev-parse HEAD)" -o /build/veneur-proxy ./cmd/veneur-proxy
+CMD cp -r henson /build/ && env GOBIN=/build go install -a -v -ldflags "-X github.com/stripe/veneur.VERSION=$(git rev-parse HEAD)" ./cmd/...
```

Comment:
I suggested doing the `go install` instead of the `go build -o`; it seems to do the right thing; any objections @aditya-stripe or @cory-stripe? (: